From e7bb7c7683a3e3bd8f8731b8990a3978dce81501 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Fri, 18 Aug 2017 03:27:30 +0000
Subject: [PATCH] grub: enable serial console by default

Port this patch from grub-2.00 to grub-efi-2.02.
Have grub go to the serial console and the default VGA console.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
Signed-off-by: Guojian Zhou <guojian.zhou@windriver.com>
---
 util/grub.d/00_header.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/util/grub.d/00_header.in b/util/grub.d/00_header.in
index 93a9023..02ec1c7 100644
--- a/util/grub.d/00_header.in
+++ b/util/grub.d/00_header.in
@@ -25,6 +25,10 @@ grub_lang=`echo $LANG | cut -d . -f 1`
 export TEXTDOMAIN=@PACKAGE@
 export TEXTDOMAINDIR="@localedir@"
 
+export GRUB_TERMINAL_INPUT="console serial"
+export GRUB_TERMINAL_OUTPUT="console serial"
+export GRUB_SERIAL_COMMAND="serial --speed=115200 --unit=0 --word=8 --parity=no --stop=1"
+
 . "$pkgdatadir/grub-mkconfig_lib"
 
 # Do this as early as possible, since other commands might depend on it.
-- 
2.7.4

